app/sequencescape_excel/sequencescape_excel/specialised_field/control_type.rb

```diff
@@ -19,7 +19,6 @@ def update(_attributes = {})
           sample.control = false
           sample.control_type = nil
         end
-        sample.save
```

Comment:
And this was the source of all the pain.

Reply:
So the sample was getting saved twice as part of the import process?

Reply:
Sorry read the commit message now :)

Reply:
So looks like they all get saved in `update_sample` in `Row.rb`

Reply:
Yep! Which should also improve speeds a little, as we'll have a single write per sample.